php56/provision.sh

```diff
@@ -65,16 +64,28 @@ package_install() {
 
 configure() {
   # Copy nginx configuration from local
-  cp "${DIR}/php5.6-upstream.conf" "/etc/nginx/upstreams/php56.conf"
+  cp -f "${DIR}/php5.6-upstream.conf" "/etc/nginx/upstreams/php56.conf"
+  echo " * Copied ${DIR}/php5.6-upstream.conf              to /etc/nginx/upstreams/php56.conf"
 
+g
   # Copy php-fpm configuration from local
-  cp "${DIR}/php5.6-fpm.conf" "/etc/php/5.6/fpm/php-fpm.conf"
-  cp "${DIR}/php5.6-www.conf" "/etc/php/5.6/fpm/pool.d/www.conf"
-  cp "${DIR}/php5.6-custom.ini" "/etc/php/5.6/fpm/conf.d/php-custom.ini"
-  cp "/srv/config/php-config/opcache.ini" "/etc/php/5.6/fpm/conf.d/opcache.ini"
-  cp "/srv/config/php-config/xdebug.ini" "/etc/php/5.6/mods-available/xdebug.ini"
+  cp -f "${DIR}/php5.6-fpm.conf" "/etc/php/5.6/fpm/php-fpm.conf"
+  echo " * Copied ${DIR}/php5.6-fpm.conf                   to /etc/php/5.6/fpm/php-fpm.conf"
+
+  cp -f "${DIR}/php5.6-www.conf" "/etc/php/5.6/fpm/pool.d/www.conf"
+  echo " * Copied ${DIR}/php5.6-www.conf                   to /etc/php/5.6/fpm/pool.d/www.conf"
+
+  cp -f "${DIR}/php5.6-custom.ini" "/etc/php/5.6/fpm/conf.d/php-custom.ini"
+  echo " * Copied ${DIR}/php5.6-custom.ini                 to /etc/php/5.6/fpm/conf.d/php-custom.ini"
+
+  cp -f "/srv/config/php-config/opcache.ini" "/etc/php/5.6/fpm/conf.d/opcache.ini"
+  echo " * Copied /srv/config/php-config/opcache.ini       to /etc/php/5.6/fpm/conf.d/opcache.ini"
+
+  cp -f "/srv/config/php-config/xdebug.ini" "/etc/php/5.6/mods-available/xdebug.ini"
+  echo " * Copied /srv/config/php-config/xdebug.ini        to /etc/php/5.6/mods-available/xdebug.ini"
+
   if [[ -e /srv/config/php-config/mailcatcher.ini ]]; then
-    cp "/srv/config/php-config/mailcatcher.ini" "/etc/php/5.6/mods-available/mailcatcher.ini"
+    cp -f "/srv/config/php-config/mailcatcher.ini" "/etc/php/5.6/mods-available/mailcatcher.ini"
```

Comment:
We still require that code? we arenot supporting anymore mailcatcher

Reply:
this repo gets cloned by some pretty old versions of VVV, though we should probably start cleaning up mailcatcher configs actually

Reply:
also because they are old version of php the new php version doesn't have mailcatcher at all, so just for consistency we can remove it.

Reply:
usually, if mailhog is present then mailcatcher can be removed, otherwise I'd rather not break stuff for people who haven't updated.

Instead lets put a note that they should upgrade VVV

Reply:
no all the php version add mailcatcher but I don't think that we should support also because if VVV is old maybe the apt package is not supported on their VM.

Reply:
oooh, no, if the file exists it will, but that file is in VVV core and doesn't exist in newer versions, so it'll copy it but the file is only present on old VVV versions.

When we changed to MailHog we deleted the mailcatcher config, so in newer versions it isn't present